setup/src/Magento/Setup/Model/Installer.php

```diff
@@ -925,7 +925,7 @@ private function throwExceptionForNotWritablePaths(array $paths)
      */
     private function handleDBSchemaData($setup, $type, array $request)
     {
-        if (!(($type === 'schema') || ($type === 'data'))) {
+        if (!$type === 'schema') || ($type === 'data')) {
```

Comment:
Operator order is the following: `!$type` converts it to `bool(false)` and further comparison `false` and `'schema'`
This condition will always return false

Reply:
I just forgot to add one `(`, see the other review comment ;-)

I will fix this one.